src/ESPEasyEth.ino

```diff
@@ -0,0 +1,90 @@
+
+#ifdef HAS_ETHERNET
+  // #define ETH_CLK_MODE ETH_CLOCK_GPIO17_OUT
+  // #define ETH_PHY_POWER 12
+  #include "ETH.h"
+  #include <esp_eth.h>
+
+String EthGetHostname()
+{
+  String hostnameToReturn(Settings.getHostname());
+  hostnameToReturn.replace(" ", "-");
+  hostnameToReturn.replace("_", "-"); // See RFC952
+  return hostnameToReturn;
+}
```

Comment:
Why do you need this extra function? It looks like a function we already have to replace chars in a hostname.

Reply:
We do have such a function?
Which one? Only i can found was in ESPEasyEth.ino but it was strictly related to WiFi

Reply:
Well the same rules apply to the allowed characters.
So if it is no longer limited to WiFi, then move it to Networking.ino.
Better not do copy/paste of code or else people here may confuse my allergy for code duplication with another virus outbreak...